### ISSUE TITLE
Freeze BiSeNetv2 batch normalization layers during fine-tuning

### ISSUE THREAD

**Issue by himanshunaidu** [himanshunaidu/ML_Pipeline_iOSPointMapper#33]
In the current PyTorch implementation of BiSeNetv2, there is no option to freeze the batch normalization layers during fine-tuning. 

Can take reference from EdgeNets:

```
if args.freeze_bn:
        print_info_message('Freezing batch normalization layers')
        for m in model.modules():
            if isinstance(m, nn.BatchNorm2d):
                m.eval()
                m.weight.requires_grad = False
                m.bias.requires_grad = False
```
